netstandard/ref/System.Reflection.Emit.cs

```diff
@@ -4,6 +4,187 @@
 
 namespace System.Reflection.Emit
 {
+    public sealed partial class AssemblyBuilder : System.Reflection.Assembly
+    {
+        internal AssemblyBuilder() { }
+        public override string FullName { get { throw null; } }
+        public override bool IsDynamic { get { throw null; } }
+        public override System.Reflection.Module ManifestModule { get { throw null; } }
+        public static System.Reflection.Emit.AssemblyBuilder DefineDynamicAssembly(System.Reflection.AssemblyName name, System.Reflection.Emit.AssemblyBuilderAccess access) { throw null; }
+        public static System.Reflection.Emit.AssemblyBuilder DefineDynamicAssembly(System.Reflection.AssemblyName name, System.Reflection.Emit.AssemblyBuilderAccess access, System.Collections.Generic.IEnumerable<System.Reflection.Emit.CustomAttributeBuilder> assemblyAttributes) { throw null; }
+        public System.Reflection.Emit.ModuleBuilder DefineDynamicModule(string name) { throw null; }
+        public override bool Equals(object obj) { throw null; }
+        public System.Reflection.Emit.ModuleBuilder GetDynamicModule(string name) { throw null; }
+        public override int GetHashCode() { throw null; }
+        public override System.Reflection.ManifestResourceInfo GetManifestResourceInfo(string resourceName) { throw null; }
+        public override string[] GetManifestResourceNames() { throw null; }
+        public override System.IO.Stream GetManifestResourceStream(string name) { throw null; }
+        public void SetCustomAttribute(System.Reflection.ConstructorInfo con, byte[] binaryAttribute) { }
```

Comment:
Do we want to use `con` name ?

Reply:
Better use `info` instead, as it is more informative (pun intended).

Reply:
If the name has already shipped in Core, surely we can't change it. Either we'd have to leave it the same way in Core, making the same piece of code work for Core or fail in Standard, or view versa... or change Core as well, making it consistent going forward, but breaking existing code. I suspect that living with a suboptimal name is better than either of these.

(I think we *may* have had a discussion around whether or not parameter names can be changed, but I can't see it now. @terrajobst it might be worth clarifying that.)

Reply:
So parameter name are part of the signature in the standard? That's a bit unfortunate in my opinion.

Reply:
They have to be, due to named arguments. If I call `foo.SetCustomAttribute(con: null, null)` it would be *awful* if that compiled on some platforms but not on others.

Reply:
As @jskeet said, arguments, while not required for binary compatibility [1], are part of source compatibility.

That being said, we unfortunately don't have a 100% alignment of parameter names between different .NET implementations. We can, in theory, have different parameter names in .NET Standard but it would make it harder for people to use multi-targeting (which is compiling the same code base for different frameworks) as they (might) now have to updated named arguments.

[1] Of course, folks can use reflection to depend on parameter names, but I'd consider this outside of what we normally refer to as binary compatibility.

Reply:
#wont-fix